src/pkg/retention/controller.go

```diff
@@ -179,26 +184,55 @@ func (r *DefaultAPIController) DeleteRetention(id int64) error {
 		}
 	}
 
+	ctx := orm.Context()
```

Comment:
update the method to give the context as the first parameter?

Reply:
I will change this in future PR when refactor the API with swagger